mozilla-central/setup

```diff
@@ -266,10 +266,12 @@ if [ -n "$INDEXED_GIT_REV" ]; then
 
     # Special case: xptdata.cpp is a giant file and is different for each platform, but
     # the differences are not particularly relevant so let's just keep the Linux one.
-    rm -f              generated-macosx64/xpcom/reflect/xptinfo/xptdata.cpp
-    rm -f                 generated-win64/xpcom/reflect/xptinfo/xptdata.cpp
-    rm -f analysis-macosx64/__GENERATED__/xpcom/reflect/xptinfo/xptdata.cpp
-    rm -f    analysis-win64/__GENERATED__/xpcom/reflect/xptinfo/xptdata.cpp
+    rm -f                   generated-macosx64/xpcom/reflect/xptinfo/xptdata.cpp
```

Comment:
Maybe we should write this as a loop?

Reply:
Good point, fixed. Not sure why I didn't think of that...